### ISSUE TITLE
Craft that was interrupted due to lack of tool charges asks for the tool repeatedly at resumption

### ISSUE THREAD

**Issue by l29ah** [CleverRaven/Cataclysm-DDA#32573]
# Steps To Reproduce

1. Load the game
2. Resume crafting of the forge at the table
3. Select soldering iron
4. The game will ask you again

[2019.07.20.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/3422754/2019.07.20.zip)

0.D-5805-g50eb478eb0-dirty

**Comment by TechyBen** [comment]
I can confirm this too.

**Comment by berangus** [comment]
I've had this issue twice, and both times it seemed to be triggered by starting a craft using a tool with insufficient charges, and then resuming the craft with a different tool -- although it would probably happen after reloading the same tool as seems to be implied by OP.

The first time I encountered this bug, I still had two available tools to choose from when resuming the craft. I had to select which tool to use every in-game turn (holding down the button for a few IRL minutes got it done). The second time I encountered this bug, after my first tool ran out of charges I had only one available tool left. This caused the crafting to proceed automatically, but incredibly slowly in real-time. I suspect the game was triggering the "which tool to use?" question, and then automatically selecting the only available tool, every in-game turn. I didn't get any pop-up to reselect the tool this time but I think it's the same bug.
